.github/workflows/cd.yml

```
@@ -15,9 +15,35 @@ on:
 jobs:
+  upgrade:
+    name: Upgrade Helm Chart Versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          filter: tree:0
+      - name: Get Release Tag
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:2:48: Double quote to prevent globbing and word splitting

(shellcheck)